package.json

```diff
@@ -5,22 +5,28 @@
   "main": "index.js",
   "scripts": {
     "start": "node ./index.js",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/.bin/lab -v -c --timeout 5000"
   },
   "author": "Laurie Voss <laurie@npmjs.com>",
   "license": "ISC",
   "dependencies": {
-    "mysql": "~2.0.1",
-    "hapi": "~2.3.0",
-    "manta": "~1.2.6",
     "assert": "~1.1.1",
-    "moment": "~2.5.1",
     "async": "~0.2.10",
-    "manta-client": "~1.0.5",
+    "compact-uuid": "0.0.0",
```

Comment:
are you locked on 0.0.0 for a reason? It looks like 0.0.1 is a thing: https://www.npmjs.com/package/compact-uuid